addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1770,7 +1770,6 @@ body.editor_enable.editor_has_snippets {
         }
         &.oe_active {
             display: block;
-            z-index: 1;
```

Comment:
Code updated as per your last review.
The issue I had now that we show 2 o_handles is that the arrow (to resize) from the parent one were bellow the o_handles of the image.
This is enough to fix the issue, see 
![image](https://user-images.githubusercontent.com/30048408/117705463-5a380e80-b1cc-11eb-86c7-7ddb547c1e5e.png)

If we want to keep that, it seems like the overlay from the image would need to be the first in the DOM, but this is not easy as Editor are constructed recursively in auto-parent discovery, it is just not about inversing an array or something.

I blamed the whole history of this line, introduced [here](https://github.com/odoo/odoo/commit/4f27e52cabb77b8b1a9637a11185ddf882adc9af#diff-f1acdd04893b739b62e26ae33945d012c4ce8df627f23ea84968f707369c8eaeR458) and modified [here](https://github.com/odoo/odoo/commit/623d7358d841f70fd8d916b8cbe80d08a253d33a#diff-f1acdd04893b739b62e26ae33945d012c4ce8df627f23ea84968f707369c8eaeL460)

Not sure if this z-index was still needed after those changes.

Reply:
It is not, since we don't have an array :) they "are constructed recursively in auto-parent discovery"
https://github.com/odoo-dev/odoo/blame/f9f07cbcfdd0608ebac24fa9eb0cbb10fbef996b/addons/web_editor/static/src/js/editor/snippets.editor.js#L2061

Reply:
PR Updated